navigation/navigation_launch/launch/navigation.launch

```diff
@@ -1,6 +1,45 @@
 <?xml version="1.0" ?>
 <launch>
- <!-- <include file="$(find imu_complementary_filter)/launch/complementary_filter.launch"/> -->
-  <include file="$(find underwater_odom)/launch/underwater_odom.launch"/>
-  <include file="$(find robot_localization)/launch/dvl_ekf_template.launch"/>
+
+  <!-- Navigation for real AUV -->
+  <group if="$(eval type == 'real')">
+
+    <!-- LOS guidance -->
+    <node pkg="los_guidance" type="los_guidance_backstepping.py" name="los_guidance" />
```

Comment:
I think guidance nodes are more at home in motion, even if there is an element on pathplanning in LOS

Reply:
I think that makes sense, yeah. Commits fadbaa1 and aa8b3c5 address the move, although this will be expanded upon once the guidance and controller are separated.